lib/fizzy/parser.cpp

```diff
@@ -97,6 +97,12 @@ inline parser_result<ConstantExpression> parse_constant_expression(
             result.value.constant = static_cast<uint64_t>(value);
             break;
         }
+        case Instr::f32_const:
```

Comment:
@gumb0 these are silently unhandled. Not sure how to handle these nicely.

Reply:
From the point of view of running the spec tests it seems fine to ignore them here (I would zero out `result.value.constant` to make it deterministic)

The test accessing the floating point global would either be skipped if it uses some float opcode, or it will return the value of the global directly, then it will be skipped as the test expecting float result value.

But outside of tests this would mean that accessing such global directly or calling a function returning it will just silently return 0.